jetty-distribution/src/main/resources/etc/fileserver.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0"?>
+<!DOCTYPE Configure PUBLIC "-//Jetty//Configure//EN" "http://www.eclipse.org/jetty/configure.dtd">
+
+<Configure id="Server" class="org.eclipse.jetty.server.Server">
+
+  <Set name="handler">
+    <New class="org.eclipse.jetty.server.handler.HandlerList">
```

Comment:
as a module, it should not build it's onw server list but use the one built by the server module.   It should then add a context and a resource handler within that context